server/services/transactionEventsController.go

```diff
@@ -196,6 +196,89 @@ func (controller *transactionEventsController) extractEventsESDTWipe(tx *transac
 	return typedEvents, nil
 }
 
+func (controller *transactionEventsController) extractEventsESDTNFTCreate(tx *transaction.ApiTransactionResult) ([]*eventESDT, error) {
+	rawEvents := controller.findManyEventsByIdentifier(tx, transactionEventESDTNFTCreate)
+	typedEvents := make([]*eventESDT, 0, len(rawEvents))
+
+	for _, event := range rawEvents {
+		numTopics := len(event.Topics)
+		if numTopics != 4 {
```

Comment:
could have extracted 4 into a constant, perhaps numTopicsESDTNFTCreate

Reply:
Indeed. Fixed (here and below).